src/main/java/ui/screens/BurndownScreen.java

```diff
@@ -75,13 +75,33 @@ protected void afterVisualizationMount() {
             updateDisplayedSprints(overlayCheckBox.isSelected());
         });
 
+        sprint_name.textProperty().bind(Bindings.createStringBinding(() -> {
+            List<Sprint> sprints = sprintMultiselect.getSelectionModel().getSelectedItems();
+            String sprint = "";
+            if (sprints.size() == 0) {
+                sprint = "(No Sprint Selected)";
```

Comment:
Can just immediately return this value, or in conjunction with the comment below, use a ternary to evaluate what to return based on the selected sprint count, either works.

Reply:
Think I did this